src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -80,6 +94,22 @@ export const DolphinSettings: React.FC<{ dolphinType: DolphinLaunchType }> = ({
           />
         </SettingItem>
       </DevGuard>
+      {dolphinType === DolphinLaunchType.NETPLAY && (
+        <SettingItem name="Play Button Functionality">
+          <RadioGroup value={launchMeleeOnPlay} onChange={(event) => onLaunchMeleeChange(event)}>
+            <FormControlLabel
+              value={true}
+              label="Launch Melee when clicking play on the home screen"
+              control={<Radio />}
+            />
+            <FormControlLabel
+              value={false}
+              label="Launch Dolphin when clicking play on the home screen"
+              control={<Radio />}
+            />
+          </RadioGroup>
+        </SettingItem>
+      )}
```

Comment:
I don't really think this belongs here. I think it should one of the first options in the Game & Replays settings menu. I think I would prefer to keep the netplay and playback dolphin settings pretty much the same.